src/Compilers/CSharp/Test/Symbol/Symbols/Source/EventTests.cs

```diff
@@ -2842,20 +2858,20 @@ static void validate(ModuleSymbol module)
                 AssertEx.Equal(["A(1)", "A(5)"], e.GetAttributes().ToStrings());
                 AssertEx.Equal([.. compiledGeneratedAttr, "A(2)"], e.AddMethod!.GetAttributes().ToStrings());
                 AssertEx.Equal(["A(3)"], e.AddMethod.Parameters.Single().GetAttributes().ToStrings());
```

Comment:
I am confused that this attribute is still here, I thought we were giving a warning that it was ignored now.

Reply:
That's a pre-existing issue. Although we warn that it's ignored, we don't ignore it. See https://github.com/dotnet/roslyn/issues/77254.